src/index.js

```diff
@@ -21,10 +28,20 @@ const render = () => {
     const label = document.createElement('label');
     label.classList.add('task-label');
 
+    const rmIcon = document.createElement('i');
+    rmIcon.classList.add('fas');
+    rmIcon.classList.add('fa-trash-alt');
+
+    const editIcon = document.createElement('i');
+    editIcon.classList.add('fas');
+    editIcon.classList.add('fa-edit');
+
     label.innerHTML = task.name;
 
     li.append(checkbox);
     li.append(label);
+    li.append(rmIcon);
+    li.append(editIcon);
     taskList.appendChild(li);
 
     checkbox.addEventListener('change', (e) => {
```

Comment:
- [ ] When I check a todo item it gets checked and it also updates the local storage completed attribute properly. But after I refresh the page the DOM/view gets back to the default state (the checked todo gets unchecked in the DOM). Kindly fix this issue. :pray: 

![Screenshot from 2021-12-05 01-08-25](https://user-images.githubusercontent.com/64894927/144726101-1bba337a-f2b6-4878-bc8b-055134077d46.png)